sysding

```diff
@@ -196,13 +245,23 @@ setup_route() {
    typeset def_tgt
    typeset def_dst
    typeset rt
+   typeset ipvers
   
    def_tgt=$1
    def_dst=$2
 
    rt=0
 
-   cmd_out=$( route -p add ${def_tgt} ${def_dst} 2>&1 )
+   # https://github.com/olbohlen/sysding/issues/1
+   # check if def_addr is IPv6 and force the -inet6 flag for route(1M)
+   echo ${def_dst} | egrep ":[0-9]*:" >/dev/null 2>&1
```

Comment:
what about an address that consists only from letters? [a-fA-F] should be added as well.